boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java

```diff
@@ -137,4 +141,55 @@ private void writeSpringBootDockerFile(File dockerFile, String libertyImage) thr
                 + "/config/dropins/spring/thin-${APP_FILE}");
         Files.write(dockerFile.toPath(), lines, Charset.forName("UTF-8"));
     }
+
+    public void createDockerIgnore() throws Exception {
+        if (springBootVersion != null) {
+            createSpringBootDockerIgnore();
+        } else {
+            throw new MojoExecutionException("Unable to create a .dockerignore file because application type is not supported");
+        }      
+    }
+
+    private void createSpringBootDockerIgnore() throws IOException {
+        File dockerIgnore = new File(projectDirectory, ".dockerignore");
+        boolean dockerIgnoreAlreadyExists = false;
+        try {
+            Files.createFile(dockerIgnore.toPath());
+            log.info("Creating .dockerignore: "+ dockerIgnore.getAbsolutePath());          
+        } catch (FileAlreadyExistsException e) {
+            log.warn("The .dockerignore file already exists");
+            dockerIgnoreAlreadyExists = true;
+        }
+        writeDockerIgnore(dockerIgnore, dockerIgnoreAlreadyExists);
+    }
+
+    private void writeDockerIgnore(File dockerIgnore, boolean dockerIgnoreAlreadyExists) throws IOException {
+        ArrayList<String> lines = new ArrayList<>();
+        if (pluginAlreadyExecuted(dockerIgnore)) {
+            return;
+        }
+
+        //Ignore the build.log getting generated by integration tests and the liberty runtime folder
+        if (dockerIgnoreAlreadyExists && dockerIgnore.length() > 0) {
+            lines.add("\n");
```

Comment:
Might be OK but just to be safe replace "\n" with `System.getProperty("line.separator")`